lib/hooks/useBox/box.treat.ts

```diff
@@ -1,22 +1,10 @@
-import mapValues from 'lodash/mapValues';
 import { css, Styles, style } from 'sku/treat';
 import { Properties } from 'csstype';
 import { darken, lighten } from 'polished';
-import { getLightVariant } from '../../atoms/utils/a11y';
-import isLight from '../../atoms/utils/isLight';
-import { px } from '../../atoms/utils/toUnit';
-
-const mapToCssRule = <Map extends string, Value extends string | number>(
-  map: Record<Map, Value>,
-  propertyName: keyof Properties,
-  mapper?: (value: Value, propertyName: keyof Properties) => Styles,
-) => {
-  type MappedStyles = Record<Map, Styles>;
-
-  return mapValues(map, (value: Value) =>
-    mapper ? mapper(value, propertyName) : { [propertyName]: value },
-  ) as MappedStyles;
-};
+import { getLightVariant } from '../../utils/a11y';
+import isLight from '../../utils/isLight';
+import { px } from '../../utils/toUnit';
+import mapToStyleProperty from '../../utils/mapToStyleProperty';
```

Comment:
What are your thoughts on having a central `util` export that gives you all of our utils? I reckon `import { px, getLightVariant, isLight, px, mapToStyleProperty } from '../../utils';` would be easier to handle/parse.

Reply:
I have a few stacked pieces of work, are you happy if we do this as a refactor after the migration work?